pkg/chartutil/values.go

```diff
@@ -388,7 +388,13 @@ func istable(v interface{}) bool {
 	return ok
 }
 
-// PathValue takes a yaml path with . notation and returns the value if exists
+// PathValue takes a path that traverses a YAML structure and returns the value at the end of that path.
+// The path starts at the root of the YAML structure and is comprised of YAML keys separated by periods.
+// Given the following YAML data the value at path "chapter.one.title" is "Loomings".
+//
+// 		chapter:
```

Comment:
The comment is excellent. My only hesitation is the generous indentation of the example document fragment. It's hard to tell reading it in GitHub, but it looks like a space and two tabs. I'm not sure how much indentation it takes to get Godoc to render the fragment as pre-formatted text.